src/views/resultsView/createScanProvider.ts

```diff
@@ -137,6 +137,18 @@ async function doesBranchMatch(context: vscode.ExtensionContext, logs: Logs) {
   }
 }
 
+async function doesProjectMatch(context: vscode.ExtensionContext, logs: Logs) {
+  const projectForScan: Item = context.workspaceState.get(constants.projectIdKey);
```

Comment:
add try-catch to this logic
think about case of none SCM project

Reply:
does not need, this function does not thrown error. it returns undefined if there is a problem

Reply:
so if it's undefine we can stop and return here because it's not SCM project
no?